pandas/tests/series/common.py

```diff
@@ -0,0 +1,30 @@
+from pandas.util.decorators import cache_readonly
+import pandas.util.testing as tm
+import pandas as pd
+
+_ts = tm.makeTimeSeries()
+
+
+class TestData(object):
+
+    @cache_readonly
+    def ts(self):
+        ts = _ts.copy()
+        ts.name = 'ts'
+        return ts
+
+    @cache_readonly
+    def series(self):
+        series = tm.makeStringSeries()
+        series.name = 'series'
```

Comment:
Oops, good point; to be on the safe side we should clear() the `_cache` variable (set by `cache_readonly`) in between unit tests. same with the DataFrame tests